js/ui/range_slider.js

```diff
@@ -195,8 +195,9 @@ const RangeSlider = Slider.inherit({
 
         this._changeValueOnSwipe(newRatio);
 
-        const startValue = this.option('start');
-        const endValue = this.option('end');
+        const startValue = this._getActualStartValue() || this.option('start');
```

Comment:
Shouldn't we use nullish coalescing operator `??` instead of `||` in all such places? Seems like this code does not work if `_getActualStartValue` returns `0`.